tests/test_peekExposureTask.py

```diff
@@ -0,0 +1,104 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 AURA/LSST.
```

Comment:
This is the old preamble.

Reply:
Huh, it's probably like that in loads of these files then (I just copied and pasted from the quickFrameMeasurement tests as a starting point).

Reply:
It is outdated there as well.